F21/maddiekoehler/lower.py

```diff
@@ -0,0 +1,40 @@
+### A function that lowercases all letters in a word
+###	parameters:
+###		string: word
+###		string: lang
+###
+
+
+def makeLower(word, lang):
+
+	#Turkish or Azerbaijani special characters
+	if 'tr' in lang or 'az' in lang:
+		word = word.replace('I','ı')
+		word = word.lower()
+
+	#Irish special characters
+	irishVowels = ['A','E','I','O','U','Á','É','Í','Ó','Ú']
+	if 'ga' in lang and len(word)>1:
+		if word[0] == 'n' or word[0] == 't':
+			if word[1] in irishVowels:
```

Comment:
Pass   Expected: nõg   Actual: nõg
Fail   Expected: nõg   Actual: n-õg 

Both words are the same but different results. again I don't know why this happens, code seems to be correct